MAP.java

```diff
@@ -0,0 +1,485 @@
+import java.io.*;
+import java.util.Random;
+
+public class MAP {
+
+	private static String ERRO;
+	private static int food1 = 0;
+	private static int hunger = 10, hungerF = 0;
+	public static void main (String[] args){
+		Random generator = new Random();
+
+		BufferedReader input;
+		input = new BufferedReader(new InputStreamReader(System.in));
+		String[][] map = new String [17][17];
+		String[][] monster = new String [17][17];
+		int i;
+		int j;
+
+		int T = 0;
+		float life = 100f;
+
+		int hungerV = 0;
+
+
+
+		boolean begin = false;
+
+		try{
+
+			generator.nextInt(10);
+			int random;
+
+			//=======Monstro_Cachorro_Dormindo=======//
+
+			highWallWalk(map, 0, 17, "Muralha lisa, extremamente alta", 16);
+
+
+			highWallWalk(map, 1, 16, "Gramado, pouca iluminacao", 15);
+
+			for(i = 2 ;i < 15 ;i++){
+				for(j = 2 ;j < 15 ;j++){
+					map[i][j] = "Gramado, floresta escura";
+				}
+			}
+			int u = 3;
+			int v = 1;
+			map[u][v] = "Gramado, porem bem iluminado";
+
+			u = 4;
+			v = 4;
+
+			GameScreenCli.printExplorerString("				EXPLORER v.0");
+			System.out.println();
+			System.out.println("		Bem vindo ao EXPLORER v.0, ");
+			String L_MAPA = ("Seu local e: "+ v +" ; "+ u +" .");
+			String leftHand = ("Mao esquerda vazia.");
+			String rightHand = ("Mao direita vazia.");
+
+			gameStatusNow(life, hunger, food1, L_MAPA, leftHand, rightHand,
+					" 	Para te manter vivo      -->		Vida: ",
+					" 	Para te manter atento    -->		Fome: ",
+					"	Para nao ficar com fome  -->	 	  ", "" +
+							" 	Para te localizar     --> 			",
+					"		<--	Quando houver comida, aparecera aqui",
+					"	Para controlar o que carrega-->			",
+					"		<--	Aqui aparecem os mostros. Cuidado...");
+
+			GameScreenCli.printRules();
+
+
```

Comment:
Just missed an empty input to give the player time to read the instructions. But I'll fix that. Congratulations on your work ^^